applications/feedback_sentiment_analyzer/DEVELOPMENT.md

```diff
@@ -0,0 +1,76 @@
+# FSA development
+
+This document describes the process for adding an additional set of AWS Lambda functions for use with the Feedback Sentiment Analyzer (FSA). The requirements for each individual Lambda function are in [SPECIFICATION.md](./SPECIFICATION.md).
```

Comment:
Not new functions as far as the reader is concerned. Same functions with new language implementation.

Reply:
```suggestion
This document explains how to implement the AWS Lambda functions detailed in [SPECIFICATION.md](./SPECIFICATION.md#lambda-functions).
```

Reply:
Implement isn't the correct word here, maybe integrate?
```suggestion
This document explains how to integrate the AWS Lambda functions detailed in [SPECIFICATION.md](./SPECIFICATION.md#lambda-functions) in your SDK of choice.
```